roles/cs.elasticsearch/templates/elasticsearch.yml

```diff
@@ -7,22 +7,41 @@
 
 cluster.name: {{ elasticsearch_cluster_name }}
 {% if elasticsearch_version_number is version('7.0.0', '>=') %}
-cluster.initial_master_nodes: ["{{ elasticsearch_node_name }}"]
+discovery.type: single-node
```

Comment:
this is not supported in elasticsearch 7

Reply:
It's working since 7.0.0